avocado/core/remote/runner.py

```diff
@@ -192,6 +196,7 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
         if not timeout:     # avoid timeout = 0
             timeout = None
         summary = set()
+        warning_message = "VERSIONS   : Local %s and remote %s are not the same."
```

Comment:
Just for the record, this is not C. We don't have to define our variables at the beginning so we can only produce this `warning_message` where it's needed and only when it's needed. Anyway it's just a note for the future 😉.